src/azul/queues.py

```diff
@@ -36,16 +36,13 @@ def __init__(self, delete=False, json_body=True):
 
     def list(self):
         logger.info('Listing queues')
-        print('\n{:<35s}{:^20s}{:^20s}{:^18s}\n'.format('Queue Name',
-                                                        'Messages Available',
-                                                        'Messages in Flight',
-                                                        'Messages Delayed'))
+        print(f'\n{"Queue Name":<35s}{"Messages Available":^20s}{"Messages In Flight":^20s}{"Messages Delayed":^18s}\n')
```

Comment:
Could we break this line up into four so it correlates more to the `print` statement in the loop?